include/tester/ssp/SoftphoneManager.hpp

```diff
@@ -10,8 +10,8 @@
 class SoftphoneManager
 {
 public:
-    SoftphoneManager(int number_of_softphones, int port, std::string & PBX_IP):
-        _PORT(port), _PBX_IP(PBX_IP)
+    SoftphoneManager(int number_of_softphones, int port, std::string domain):
+        _port(port), _domain(domain)
```

Comment:
you're passing the value of `domain` into the constructor of `_domain`. read about `std::move` and use it accordingly.